src/App.jsx

```diff
@@ -1,11 +1,147 @@
-
+import {useState} from "react";
+import Button from './components/Button/Button.jsx'
+import FruitCounter from "./components/FruitCounter/FruitCounter.jsx";
+import InputField from "./components/InputField/InputField.jsx";
 import './App.css'
 
 function App() {
+  const [bananas, setBananas] = useState(0);
+  const [strawberries, setStrawberries] = useState(0);
+  const [apples, setApples] = useState(0);
+  const [kiwis, setKiwis] = useState(0);
+
+  const [formState, setFormState] = useState({
+      firstname: '',
+      lastname: '',
+      age: 0,
+      zipcode: '',
+      deliveryFrequency: 'week',
+      deliveryTimeslot: 'daytime',
+      comment: '',
+      agreeTerms: false,
+  })
+
+  function handleFormChange(e) {
+      const inputFieldName = e.target.name;
+      const inputFieldValue = e.target.type === 'checkbox' ? e.target.checked : e.target.value;
+
+      setFormState({
+          ...formState,
+          [inputFieldName]: inputFieldValue,
+      })
+  }
+
+  function resetFruits () {
+      setBananas(0);
+      setStrawberries(0);
+      setApples(0);
+      setKiwis(0);
+  }
+
+  function handleSubmit(e) {
+      e.preventDefault();
+      console.log(formState);
+      console.log(`Fruitmand bestelling - bananen: ${bananas}, aardbeiden: ${strawberries}, appels: ${apples}, kiwi's: ${kiwis}`);
+  }
 
   return (
     <>
         <h1>Fruitmand bezorgservice</h1>
+        <section className="outer-content__container fruit-counter__section">
+            <h3>Vul je fruitmand:</h3>
+            <article className="fruit-counter">
+                <h2>🍌 Bananen</h2>
```

Comment:
Deze twee elementen worden ook herhaald per fruitsoort, dus die zou je dan óók in het <FriutCounter /> component kunnen zetten. De naam zou je dan weer via een prop mee kunnen geven